packages/types/index.d.ts

```diff
@@ -309,13 +309,25 @@ export interface IFieldRichText extends IBaseField {
   value?: string
 }
 
+export interface IFieldBooleanRadio extends IBaseField {
+  type: 'boolean-radio'
+  value?: boolean // will default to null if not provided
+  options?: IFieldBooleanRadioOption[] // only can provide 2 OPTIONS if label is not yes/no
```

Comment:
As discussed, lets go with 2 props. Maybe something like this, but improve naming if you can
```suggestion
  trueOptionLabel: string
  falseOptionLabel: string
```

Reply:
ok i changed it to a tuple instead thanksss
`export type IFieldBooleanRadioOptions = [IFieldBooleanRadioOption, IFieldBooleanRadioOption]`